diff -up libusnic_verbs-1.1.0.237.rhel7u0/usnic_tools-1.0.1.237.rhel7u0/usnic_verbs_check.no-kmod libusnic_verbs-1.1.0.237.rhel7u0/usnic_tools-1.0.1.237.rhel7u0/usnic_verbs_check
--- libusnic_verbs-1.1.0.237.rhel7u0/usnic_tools-1.0.1.237.rhel7u0/usnic_verbs_check.no-kmod	2015-01-23 16:03:36.126338668 -0500
+++ libusnic_verbs-1.1.0.237.rhel7u0/usnic_tools-1.0.1.237.rhel7u0/usnic_verbs_check	2015-01-23 16:18:29.030448641 -0500
@@ -27,7 +27,7 @@
 # programs to be able to run successfully.
 
 need_enic_verrel=2.1.1.46
-cisco_ompi_name=openmpi-cisco
+cisco_ompi_name=openmpi
 sys_infin_root=/sys/class/infiniband
 enic_arch='.x86_64'
 usnic_verbs_arch='.x86_64'
@@ -65,8 +65,8 @@ detect_distro()
 {
     if [[ -e /etc/redhat-release ]]; then
         distro="rhel"
-        enic_module_name="kmod-enic"
-        enic_ver_separator='-'
+        enic_module_name=""
+        enic_ver_separator=''
     elif [[ -e /etc/SuSE-release ]]; then
         distro="sles"
         enic_module_name="cisco-enic-kmp-default"
@@ -96,7 +96,7 @@ verrel_to_int()
 
 print_versions()
 {
-    print_enic_version
+#    print_enic_version
     print_usnic_verbs_version
     print_libusnic_verbs_version
     print_ompi_version
@@ -151,24 +151,28 @@ print_enic_version()
 
 print_usnic_verbs_version()
 {
-    rpm=$(rpm -q kmod-usnic_verbs 2> /dev/null)
-    [[ $? != 0 ]] && rpm=""
+#    rpm=$(rpm -q kmod-usnic_verbs 2> /dev/null)
+#    [[ $? != 0 ]] && rpm=""
 
     usnic_verbs=$(lsmod | grep ^usnic_verbs)
     usnic_verbs_vers=$(modinfo usnic_verbs 2> /dev/null | grep ^version: | awk '{ print $2 }')
-
-    if [[ -z "${rpm}" ]]; then
-        warn "kmod-usnic_verbs RPM is not installed"
+    if [ -n "$usnic_verbs_vers" ]; then
+	echo "usnic_verbs version ${usnic_verbs_vers}"
     else
-        rpmverrel=${rpm#kmod-usnic_verbs-}
-        rpmverrel=${rpmverrel%${usnic_verbs_arch}}
-        rpmver=${rpmverrel%-*}
-
-        echo "usnic_verbs RPM version ${rpmverrel} installed"
-
-        if [[ "${usnic_verbs_vers}" != "${rpmver}" ]]; then
-            warn "modinfo usnic_verbs version (${usnic_verbs}) does not match RPM version (${rpmver})"
-        fi
+	warn "usnic_verbs kernel module not found"
+    fi
+#    if [[ -z "${rpm}" ]]; then
+#        warn "kmod-usnic_verbs RPM is not installed"
+#    else
+#        rpmverrel=${rpm#kmod-usnic_verbs-}
+#        rpmverrel=${rpmverrel%${usnic_verbs_arch}}
+#        rpmver=${rpmverrel%-*}
+
+#        echo "usnic_verbs RPM version ${rpmverrel} installed"
+
+#        if [[ "${usnic_verbs_vers}" != "${rpmver}" ]]; then
+#            warn "modinfo usnic_verbs version (${usnic_verbs}) does not match RPM version (${rpmver})"
+#        fi
 
         if [[ -z "${usnic_verbs}" ]]; then
             iommu=$(dmesg | grep "IOMMU required but not present")
@@ -177,13 +181,13 @@ print_usnic_verbs_version()
             else
                 warn "usnic_verbs module not loaded"
             fi
-        else
-            loadver=$(cat /sys/module/usnic_verbs/version)
-            if [[ "${loadver}" != "${rpmver}" ]]; then
-                warn "Loaded usnic_verbs version (${loadver}) does not match RPM version (${rpmver})"
-            fi
+#        else
+#            loadver=$(cat /sys/module/usnic_verbs/version)
+#            if [[ "${loadver}" != "${rpmver}" ]]; then
+#                warn "Loaded usnic_verbs version (${loadver}) does not match RPM version (${rpmver})"
+#            fi
         fi
-    fi
+#    fi
 }
 
 print_libusnic_verbs_version()
